core/bcast/bcast.go

```diff
@@ -188,7 +186,7 @@ func (b Broadcaster) Broadcast(ctx context.Context, duty core.Duty, pubkey core.
 
 		err = b.eth2Cl.SubmitAggregateAttestations(ctx, []*eth2p0.SignedAggregateAndProof{&aggAndProof.SignedAggregateAndProof})
 		if err == nil {
-			log.Info(ctx, "Attestation aggregation successfully submitted to beacon node",
+			log.Info(ctx, "Successfully submitted attestation aggregation to beacon node",
```

Comment:
maybe "Successfully submitted aggregated attestation to beacon node"

Reply:
potato potato